File: coder_byte/remove_all_adjacant_matching_characters.js
function removePairs(string){
  let result = '';
  
  for(let i = 0; i < string.length; i++){
    if( i === string.length - 1){
      result += string[i];
     }
     else if(string[i] !== string[i+1]){
      result += string[i];
     }
     else{
      i += 1;
     }
  }
  
  return result;
}

File: coder_byte/remove_all_adjacant_matching_characters.py
def removePairs(string):
  str = ''
  i = 0
  
  while i < len(string):
    if i == len(string) - 1:
      result += string[i]
    elif string[i] != string[i+1]:
      result += string[i]
    else:
      i += 1
    i += 1
 return result
